src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -2652,9 +2685,26 @@ namespace HVACUnitarySystem {
                         UnitarySystem(UnitarySysNum).HeatVolumeFlowRate(Iter) /
                         UnitarySystem(UnitarySysNum).HeatVolumeFlowRate(DesignSpecMSHP(MSHPIndex).NumOfSpeedHeating);
                 }
-                UnitarySystem(UnitarySysNum).IdleVolumeAirRate = UnitarySystem(UnitarySysNum).HeatVolumeFlowRate(1);
-                UnitarySystem(UnitarySysNum).IdleMassFlowRate = UnitarySystem(UnitarySysNum).HeatMassFlowRate(1);
-                UnitarySystem(UnitarySysNum).IdleSpeedRatio = UnitarySystem(UnitarySysNum).MSHeatingSpeedRatio(1);
+            }
+            if (UnitarySystem(UnitarySysNum).CoolCoilExists) {
```

Comment:
This section can be critiqued. If a cooling coil exists, then do this. If it is multispeed then do this. If not multispeed then do this. Maybe it should be if MultiOrVarSpeedCoolCoil, that would make more sense.